cypress/e2e/util/annotationsSetup.ts

```diff
@@ -99,11 +99,15 @@ export const startEditingAnnotation = (cy: Cypress.Chainable) => {
 export const editAnnotation = (cy: Cypress.Chainable) => {
   startEditingAnnotation(cy)
 
-  cy.getByTestID('edit-annotation-message').should('have.length.of.at.least', 1)
-  cy.getByTestID('edit-annotation-message').clear()
-  cy.getByTestID('edit-annotation-message').type('lets edit this annotation...')
+  cy.getByTestID('edit-annotation-message')
+    .should('have.length.of.at.least', 1)
+    .clear()
+    .should('have.length.of.at.least', 1)
```

Comment:
I'm confused by this assertion and am hoping you can clarify.  What exactly does clear() do on the line above if the message is still of length one afterwards?

I'd have naively expected the message length to be zero since it was cleared out.

Reply:
@joelfried `clear()` empties the contents of the textbox. the `should('have.length.of.at.least', 1)` is asserting that there is at least one textbox with the testID of `edit-annotation-message` that is being rendered in the DOM. 

we don't technically need to do the `clear()` to edit the contents, but it's more representative of human behavior

Reply:
Gotcha.  I've gone with using `should('be.visible')` in my tests for similar effects.  I think the juxtaposition with the `clear()` put me in a different frame of mind.  Thanks for clarifying!